src/components/LoanCards/KvClassicLoanCardContainer.vue

```diff
@@ -345,7 +345,7 @@ export default {
 						dataObject?.data.push({
 							lenderName: action.lender.name,
 							lenderImage: action.lender?.image?.url,
-							text: `${action.lender.name} lent $${parseFloat(action.shareAmount).toFixed()}`,
```

Comment:
Should we also remove `shareAmount` from the gql query? If not isn't being used anywhere else.